cmake/px4_add_board.cmake

```diff
@@ -262,6 +266,12 @@ function(px4_add_board)
 	if(CONSTRAINED_FLASH)
 		set(px4_constrained_flash_build "1" CACHE INTERNAL "constrained flash build" FORCE)
 		add_definitions(-DCONSTRAINED_FLASH)
+		if (NO_HELP)
+			if (NOT HELP_URL)
```

Comment:
I'd remove this option and always set it to `https://docs.px4.io/master/en/modules/modules_main.html`. I don't think this will ever need to be customized (and if, we can add the option then).

Reply:
Ok.....

Reply:
Done